pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -136,14 +136,15 @@ def connection_select(context, name, **options):
     """
     Select a WBEM connection definition as current or default.
 
-    Selects a connection from the persistently stored named connections if NAME
-    exists in the store to be the current connection. If the NAME argument does
-    not exist, a list of connections from the connections definition file is
-    presented with a prompt for the user to select a connection.
-
-    Default and current connction are set if the --default option exists;
-    otherwise only the connection in the current interactive session is set .
-    Once define the default connection will be used as the server definition in
+    Selects the connection named NAME from the persistently stored named
+    connectionsto be the current connection if NAME exists in the store . If
+    the NAME argument does not exist, a list of connections from the
+    connections definition file is presented with a prompt for the user to
+    select a connection.
+
+    Default and current connection are set if the --default option exists;
```

Comment:
Change "--default option exists" to "--default option is used".

Reply:
DONEE

Reply:
I don't see this as done.

Reply:
we can fix this later as part of issue #300.